File: C/LoginProgram.c
/*Login Program CSBS-07*/
#include <stdio.h>
int main()
{
    int User_name, Pass_word;
    User_name = 123789;
    Pass_word = 987321;
    printf("Enter the Username and Password: \n");
    printf("Username : \n");
    scanf("%d", &User_name);
    printf("Password : \n");
    scanf("%d", &Pass_word);

    if(User_name == 123789 && Pass_word == 987321){
        printf("Logged in successfully");
    }
    else
        printf("Invalid Username or Password. Try again");
    return 0;
}

File: Java/login_program.java
import java.util.*;

public class login_program{

	public static void main(String[] args) {

		String username, temp_user, password, temp_pass;
		username = "Tanaj2004";
		password = "TanajisDumb";

		try (Scanner scan = new Scanner(System.in);)
		{
		System.out.println("Enter username: ");
		temp_user = scan.nextLine();
		
		System.out.println("Enter password: ");
		temp_pass = scan.nextLine();

		}
		
		if(username.equals(temp_user) == true && password.equals(temp_pass) == true) {
			System.out.print("Login Successful.");
		}		
		
		else{
			System.out.print("Invalid Credentials.");
		}
	}
}